src/support/slack/commands/onboard.js

```diff
@@ -67,41 +70,68 @@ function OnboardCommand(context) {
     const { DEFAULT_ORGANIZATION_ID: defaultOrgId } = context.env;
 
     try {
-      const [baseURLInput] = args;
-
+      const [baseURLInput, imsOrgID] = args;
       const baseURL = extractURLFromSlackInput(baseURLInput);
 
+      const flags = parseFlags(args.text || '');
+      const profileKey = flags.profile || 'default';
+      await say(`:gear: Applying profile ${profileKey}.`);
+
       if (!baseURL) {
         await say(':warning: Please provide a valid site base URL.');
         return;
       }
 
-      // see if the site was added previously
-      let site = await Site.findByBaseURL(baseURL);
+      if (!imsOrgID) {
```

Comment:
i am adding the IMS org regex in https://github.com/adobe/spacecat-shared/pull/618 - could use `Audit.IMS_ORG_ID_REGEX.test(imsOrgId)`

Reply:
this is now available for use via https://github.com/adobe/spacecat-api-service/commit/848f21d37c11c7d164e586c6e6dd9ce3c7d43ace